SampleApp/Program.cs

```diff
@@ -0,0 +1,48 @@
+using Npgsql;
+using SampleApp.db;
+using TinyHomeDataHelper;
+
+var builder = WebApplication.CreateBuilder(args);
+var services = builder.Services;
+
+services.ConfigureDataHelper<ApplicationContext, NpgsqlConnection>();
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+// Add services to the container.
+
+builder.Services.AddControllers();
+// Learn more about configuring Swagger/OpenAPI at https://aka.ms/aspnetcore/swashbuckle
+builder.Services.AddEndpointsApiExplorer();
+builder.Services.AddSwaggerGen();
+
+var app = builder.Build();

```

Comment:
I'm getting a dependency injection error when I try to run this 

Unable to resolve service for type 'Npgsql.NpgsqlConnection' while attempting to activate 'TinyHomeDataHelper.db.DapperService`1[Npgsql.NpgsqlConnection]'.